### DISCUSSION TITLE
text_sam.py  show_anns function

### DISCUSSION THREAD

**Discussion by Nomik21** [opengeos/segment-geospatial#217] [General]
Hi Professor. 

I noticed that in text_sam module in show_anns function if you define blend parameter as true, the output image is not georeferenced. On the other hand, if you define blend as False, the output grayscale image is georeferenced. 
Studying the specific code, it is in line 498. 

`if output is not None:
            if blend:
                plt.savefig(output, **kwargs)
            else:
                array_to_image(self.prediction, output, self.source)`

Can we change it in order to export  output with blend=True as a georeferenced image in order to use it with the add_raster function?
Thank you very much,

**Comment by giswqs** [comment]
Good suggestion. Can you submit a pull request?

**Reply by Nomik21** [reply]
Ok i will try. Thanks for your immediate responce.
